zone/doors.cpp

```diff
@@ -278,8 +278,11 @@ void Doors::HandleClick(Client* sender, uint8 trigger) {
 
 		/**
 		 * Key required
+		 * If using a lock_pick_item leave messaging to that code below
 		 */
-		sender->Message(Chat::LightBlue, "This is locked...");
+		if (lock_pick_item == nullptr && !IsDoorOpen()) {
+			sender->Message(Chat::LightBlue, "This is locked...");
```

Comment:
Can do if !lock_pick_item versus a direct compare to nullptr, unless 0 has a different meaning.

Reply:
So I mimic'd what the old code did when checking this variable, assuming someone there had a preference for it.

Whatever is fine with me.  Let me know.  I just didn't want to go against what they were already doing.